### PR TITLE
Create Footer

### PR DESCRIPTION
**Pull request by DanteBorges**

### CODE REVIEW

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                className="text-base text-black hover:text-white text-sm font-medium"
+              >
+                {item.name}
+              </a>
+            </div>
+          ))}
+        </nav>
+        <div className="mt-8 flex justify-center space-x-6">
+          {navigation.social.map((item) => (
+            <a
```
Comment (natandias) [inline src/components/Footer/index.js:76]: Consider using rel="noopener" on your links to improve security: https://web.dev/external-anchors-use-rel-noopener/.

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+const navigation = {
```
Comment (natandias) [inline src/components/Footer/index.js:3]: Maybe you could change this object to a separate file, to make this file a bit cleaner.

**Review by natandias — CHANGES_REQUESTED** [review]
Please, remove the .eslintcache file and put it on the .gitignore file.
There's also a bug in the mobile visualization of the footer on smaller screens.
Captura de tela de 2022-01-20 14-04-14

The contrast of the text on the footer is not good. The contrast ratio is under 4, which makes it quite unreadable. You could use white color for the text, which would improve the contrast ratio to above 6. You can also use this site: https://webaim.org/resources/contrastchecker/ to test colors contrast. 
Captura de tela de 2022-01-20 14-01-59

File: .eslintcache [hunk 3]
```diff
@@ -1 +1 @@
...
...,"column":17,"nodeType":"52","endLine":46,"endColumn":18},{"ruleId":"50","severity":1,"message":"51","line":35,"column":19,"nodeType":"52","endLine":38,"endColumn":20},{"ruleId":"53","severity":1,"message":"54","line":76,"column":13,"nodeType":"52","messageId":"55","endLine":81,"endColumn":14,"fix":"56"},"no-native-reassign",["57"],"no-negated-in-lhs",["58"],"no-unused-vars","'classNames' is defined but never used.","Identifier","unusedVar","jsx-a11y/anchor-is-valid","The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md","JSXOpeningElement","react/jsx-no-target-blank","Using target=\"_blank\" without rel=\"noreferrer\" (which implies rel=\"noopener\") is a security risk in older browsers: see https://mathiasbynens.github.io/rel-noopener/#recommendations","noTargetBlankWithoutNoreferrer",{"range":"59","text":"60"},"no-global-assign","no-unsafe-negation",[4824,4824]," rel=\"noreferrer\""]
```
Comment (emersonlopxs) [inline .eslintcache:1]: pode colocar esse arquivo `.gitignore`

File: src/components/Footer/index.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  main: [
+    { name: "Home", href: "#" },
+    { name: "About me", href: "#" },
+    { name: "Projects", href: "#" },
+    { name: "Hire me", href: "#" },
```
Comment (emersonlopxs) [inline src/components/Footer/index.js:8]: These `href`s should point something, otherwise could be removed.

File: src/components/Footer/index.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  ],
+  social: [
+ 
+    {
+      name: "Instagram",
+      href: "https://www.instagram.com/danielborggs/", rel:"noopener",
+      icon: (props) => (
+        <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
```
Comment (emersonlopxs) [inline src/components/Footer/index.js:16]: Can we move this image to a `.svg` file inside an images?

File: src/components/Footer/index.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+            clipRule="evenodd"
+          />
+        </svg>
+      ),
+    },
+    {
+      name: "Linkedin",
+      href: "https://www.linkedin.com/in/daniel-fborges/",rel:"noopener",
+      icon: (props) => (
+        <svg
```
Comment (emersonlopxs) [inline src/components/Footer/index.js:29]: Same as above

File: src/components/Footer/index.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+          <path d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"></path>
+        </svg>
+      ),
+    },
+
+    {
+      name: "GitHub",
+      href: "https://github.com/DanteBorges",rel:"noopener",
+      icon: (props) => (
+        <svg fill="currentColor" viewBox="0 0 24 24" {...props}>
```
Comment (emersonlopxs) [inline src/components/Footer/index.js:43]: Same as above

**Pull request merged**